tests/unit/test_charm.py

```diff
@@ -19,8 +20,36 @@ def test_flask_pebble_layer(harness: Harness) -> None:
     assert flask_layer == {
         "override": "replace",
         "summary": "Flask application service",
-        "command": "python3 -m gunicorn --chdir /srv/flask/app app:app -b 0.0.0.0:8000",
+        "command": "python3 -m gunicorn -c /srv/flask/gunicorn.conf.py app:app",
```

Comment:
Shouldn't `/srv/flask` and `app:app` be constants ?
Event perhaps the command line using templated string ?

Reply:
Do you mean like this?
```python
uwsgi_app_path = "app:app"
flask_base_dir = "/srv/flask"
command = f"python3 -m gunicorn -c {flask_base_dir}/gunicorn.conf.py {uwsgi_app_path}"
```

Reply:
Yeap something like this and shared with the webserver service

Reply:
Updated, but since only the `"/srv/flask"` is shared between two test cases, I just extracted that one, thanks!